admins/pageflow/user.rb

```diff
@@ -28,164 +23,273 @@ module Pageflow
     filter :email
 
     action_item(:invite, only: :index) do
-      link_to I18n.t('pageflow.admin.users.invite_user'), new_admin_user_path, :data => {:rel => 'invite_user'}
+      link_to I18n.t('pageflow.admin.users.invite_user'),
+              new_admin_user_path,
+              data: {rel: 'invite_user'}
     end
 
     show do |user|
-      div :class => :columns do
-        div do
-          attributes_table_for user do
-            row :last_name, :class => 'last_name'
-            row :first_name, :class => 'first_name'
-            row :email, :class => 'email'
-            if authorized?(:read, Account)
-              row :account, :class => 'account'
-            end
-            row :role, :class => 'role' do
-              span 'data-user-role' => user.role do
-                I18n.t(user.role, :scope => 'pageflow.admin.users.roles')
-              end
-            end
+      div do
+        attributes_table_for user do
+          row :last_name, class: 'last_name'
+          row :first_name, class: 'first_name'
+          row :email, class: 'email'
 
-            row :created_at
-            row :last_sign_in_at
-            boolean_status_tag_row :suspended?
-            row :locale do
-              I18n.t('language', locale: user.locale)
+          row :created_at
+          row :last_sign_in_at
+          boolean_status_tag_row :suspended?
+          row :locale do
+            I18n.t('language', locale: user.locale)
+          end
+          if user.admin?
+            row :admin, class: 'admin' do
+              I18n.t('pageflow.admin.users.roles.admin')
             end
           end
+        end
 
-          para do
-            link_to I18n.t('pageflow.admin.users.resend_invitation'), resend_invitation_admin_user_path(user), :method => :post, :class => 'button', :data => {:rel => 'resend_invitation'}
-          end
+        para do
+          link_to I18n.t('pageflow.admin.users.resend_invitation'),
+                  resend_invitation_admin_user_path(user),
+                  method: :post, class: 'button', data: {rel: 'resend_invitation'}
         end
 
-        panel I18n.t('activerecord.models.entry.other') do
-          embedded_index_table(user.memberships.includes(:entry).references(:pageflow_entry),
-                               blank_slate_text: I18n.t('pageflow.admin.users.empty')) do
-            table_for_collection sortable: true, class: 'memberships', i18n: Membership do
-              column :entry, sortable: 'pageflow_entries.title' do |membership|
-                link_to(membership.entry.title, admin_entry_path(membership.entry))
+        entry_memberships = user.memberships.on_entries
+                            .accessible_by(Ability.new(current_user), :index)
+        if entry_memberships.any?
+          panel I18n.t('activerecord.models.entry.other') do
+            table_for entry_memberships, class: 'memberships', i18n: Membership do
```

Comment:
Let's keep using an `embedded_index_table` here to allow for sorting and pagination.